Documentation~/CinemachineVirtualCamera.md

```diff
@@ -36,7 +36,7 @@ At any time, each Virtual Camera may be in one of these states:
 | | _Near Clip Plane_ | The closest point relative to the camera where drawing occurs. |
 | | _Far Clip Plane_ | The furthest point relative to the camera where drawing occurs. |
 | | _Dutch_ | Dutch angle. Tilts the Unity camera on the z-axis, in degrees. This property is unique to the Virtual Camera; there is no counterpart property in the Unity camera. |
-|  __Mode Override__ || Allows you to select a different camera mode to apply to the Camera component when Cinemachine activates this Virtual Camera. __Important:__ All the changes applied to the Camera component through this setting will remain after the Virtual Camera deactivation. If you set a mode override in any Virtual Camera, you should set one in all Virtual Cameras. |
+|  __Mode Override__ || Allows you to select a different camera mode to apply to the [Unity camera](https://docs.unity3d.com/Manual/class-Camera.html) component when Cinemachine activates this Virtual Camera. <br>__Important:__ All the changes applied to the Camera component through this setting will remain after the Virtual Camera deactivation. If you set a mode override in any Virtual Camera, you should set one in all Virtual Cameras. |
```

Comment:
- Thanks for adding the link, I missed this (important) detail in my last review.
- I would capitalize Unity Camera as we talk about the Camera component.
- For the line break, please use `<br />` to ensure it will be correctly interpreted by DocFX (doc build tool).

Reply:
fixed